added carousel, currently hardcoded

BeckhamL opened the pull request:
Added the carousel component. This is to check the style and positioning.

Need to add:
- Data from meal file
- Click to expand card
- Nav bar

BeckhamL committed:
added carousel, currently hardcoded

matt-amplelabs reviewed (commented):
nice! Could you add a simple example / usage block at the top the file (or in the script section) to show how to use this component? E.g.

<carousel
  v-for="meal in meals"
  v-bind:key="meal.id" 
>
</carousel>

For meal.id, use any fields you see fit (ref: https://github.com/amplelabs/amplelabs-chatbot/blob/master/data/meal.json)

Can you look into two-way binding also?
Ref: https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components

This can be another PR.

matt-amplelabs commented:
The compute() is not used properly. The way is used in the code will never be executed. Seems like you should be using one of the vue component lifecycle hooks instead (e.g. beforeMount etc.). Also the "v-bind:key="card" should be v-bind:key="card.name" (the index should be primitive type).

matt-amplelabs commented:
also, passing on the response card is a props to the component seems like a better idea also.

export default {
  name: 'carousel',
  props: ['responseCard'],
  ...

<Carousel
      v-bind:responseCard="responseCard"
      v-if="responseCard.imageUrl !== null"
    ></Carousel>

matt-amplelabs committed:
updated version in testing

Just some adj added to fetch info from UI (no meal.json). We can discuss this more.
